oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/geometry/osmtestdata/OSHDBGeometryBuilderTestOsmTestData1xxTest.java

```diff
@@ -233,14 +233,15 @@ public void test130() {
     assertTrue(result1 instanceof LineString);
     assertTrue(result2 instanceof LineString);
     assertTrue(result1.crosses(result2));
-    for (int j = 0; j < result1.getLength();j++) {
-      for (int i = 0; i < result2.getLength();i++) {
+    for (int j = 0; j < result1.getLength(); j++) {
+      for (int i = 0; i < result2.getLength(); i++) {
         assertNotEquals(((LineString) result1).getCoordinateN(j),
             ((LineString) result2).getCoordinateN(i));
       }
     }
   }
 
+  @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
the logic of the final part of this test is flawed. I think what the author tried to do is something like:

```java
    boolean hasCommonCoordinate = false;
    for (int j = 0; j < result1.getLength(); j++) {
      for (int i = 0; i < result2.getLength(); i++) {
        Coordinate c1 = ((LineString) result1).getCoordinateN(j);
        Coordinate c2 = ((LineString) result2).getCoordinateN(i);
        if (c1.equals(c2)) {
          hasCommonCoordinate = true;
        }
      }
    }
    assertTrue(hasCommonCoordinate);
```

:thinking:

Reply:
or slightly shorter:

```java
    Set<Coordinate> res1Coords = Arrays.stream(result1.getCoordinates()).collect(Collectors.toSet());
    assertTrue(Arrays.stream(result1.getCoordinates()).anyMatch(res1Coords::contains));
```

Reply:
What does this actually do? I read it as `res1Coords` is a set of `result1`s coordinates. And then the coordinates of `result1` are checked if they are inside of this set :raised_eyebrow: :thinking:

Reply:
oops, yes, the second line should be with `result2` to actually make sense. you version is what I had in mind. :bow:

Reply:
I think 1705e2cb fixed this.